scripts/install-helm.sh

```diff
@@ -195,6 +197,7 @@ cleanup_cephcsi_helm_charts() {
     fi
     "${HELM}" uninstall ${CEPHFS_CHART_NAME} --namespace ${NAMESPACE}
     "${HELM}" uninstall ${RBD_CHART_NAME} --namespace ${NAMESPACE}
+    kubectl_retry delete namespace ${NAMESPACE} --ignore-not-found
```

Comment:
Since we check for `AlreadyExists` in `kubectl_retry` for create operations [here](https://github.com/ceph/ceph-csi/pull/2377/commits/e2c64069ea26795ceb648719b567b37aed60fe66#diff-1615c06ff867567fde8b4caed7973fae157a4e384ebb043edc20cd2dd3a67bb1R17-R26), why not incorporate the `--ignore-not-found` in the `kubectl_retry` for delete operations as well?

Reply:
>Since we check for AlreadyExists in kubectl_retry for create operations here, why not incorporate the --ignore-not-found in the kubectl_retry for delete operations as well?

IMO It looks fine as it is currently, best not to over complicate the util function.

Reply:
As far as I think, It is a good addition and makes all the calls less ugly; similar to what we have done with `create` action.
Instead of writing `--ignore-not-found` in multiple delete calls, just we can mention it once in the `kubectl_retry` as we have done for `AlreadyExists`. 
@nixpanic Thoughts?

Reply:
> As far as I think, It is a good addition and makes all the future calls less ugly; similar to what we have done with `create` action.
> Instead of writing `--ignore-not-found` in multiple delete calls, just we can mention it once in the `kubectl_retry` as we have done for `AlreadyExists` for `create` action.

Sure, feel free to send a follow-up pr after this is merged. I don't see a definite need to include that in this pr.

Reply:
~~> Sure, feel free to send a follow-up pr after this is merged. I don't see a definite need to include that in this pr.~~

Added this functionality @Yuggupta27 PTAL